automated install/uninstall.sh

```diff
@@ -158,5 +204,13 @@ removePiholeFiles() {
 }
 
 ######### SCRIPT ###########
+confirmUninstall
 removeMetaPackage
```

Comment:
This should happen last since some of the dependencies may be used by the uninstall logic. This is definitely true on alpine.

Reply:
I'm starting to get the impression that a lot of things want to be uninstalled last.